crates/holochain_zome_types/src/query.rs

```diff
@@ -105,42 +111,76 @@ pub struct LinkQuery {
     pub author: Option<AgentPubKey>,
 }
 
-/// An agent's chain records, returned from an agent activity query.
+/// An agent's status and chain records returned from a `hdk::chain::get_agent_activity`
+/// query.
 #[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize, SerializedBytes)]
 pub struct AgentActivity {
-    /// Valid actions on this chain. `(sequence, action_hash)`.
+    /// Valid actions on this chain matching the filters passed to the query,
+    /// as a vector of `(sequence_index, action_hash)` tuples. **Note**:
+    /// Warrants received from other sources are not reflected in this field;
+    /// it simply contains actions whose [ `RegisterAgentActivity` ] operations
```

Comment:
In only notice this now, there shouldn't be a space between the bracket and the tick. Please review this throughout the changes.
```suggestion
    /// it simply contains actions whose [`RegisterAgentActivity`] operations
```